app/layout.tsx

```
@@ -252,6 +252,26 @@ export default function RootLayout({
             <PWAInstall />
           </div>
         </Providers>
+        
+        {/* Brevo Chat Widget */}
+        <Script
+          id="brevo-chat-widget"
+          strategy="afterInteractive"
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)